### PR TITLE
ADD minc tools install in gitactions

### PR DESCRIPTION
**Pull request by kongtiaowang**

### CODE REVIEW

**Comment by laemtl** [comment]
Amazing, TY @kongtiaowang  !

File: test/Dockerfile.test.php8 [hunk 1]
```diff
@@ -25,10 +18,50 @@ RUN apt-get install -y perl && \
...
+RUN apt-get install -y libc6 libstdc++6 imagemagick perl
+
+# Download the MINC Toolkit package
+RUN wget -P /tmp http://packages.bic.mni.mcgill.ca/minc-toolkit/Debian/minc-toolkit-1.9.18-20200813-Debian_10-x86_64.deb
+
+# Install the MINC Toolkit package
+RUN dpkg -i /tmp/minc-toolkit-1.9.18-20200813-Debian_10-x86_64.deb
+
+# Run the MINC Toolkit configuration script 
+RUN . /opt/minc/1.9.18/minc-toolkit-config.sh
```
Comment (laemtl) [inline test/Dockerfile.test.php8:58]: Will we update the config path in a separate PR?

**Review by laemtl — APPROVED** [review]

File: test/Dockerfile.test.php8 (hunk 1 above)
Comment (kongtiaowang) [reply test/Dockerfile.test.php8:58]: Yes, we will do it in a separate PR.

**Review by cmadjar — APPROVED** [review]

**Comment by cmadjar** [comment]
@kongtiaowang Could you fix the conflicts on this PR please?

**Pull request merged**